src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const dailyRate = 40;
+  let totalCost = days * dailyRate;
+
+  if (days >= 7) {
+    totalCost -= 50;
```

Comment:
Use constants for the number of days and discount values to avoid hardcoding, making the code more maintainable and readable. For example, consider using `const LONG_TERM_RENTAL = 7;` and `const LONG_TERM_DISCOUNT = 50;`.